packages/ag-charts-website/src/middleware/index.ts

```diff
@@ -1,5 +1,8 @@
+import { getIsDev } from '@utils/env';
```

Comment:
This is kind of irrelevant but I was getting Astro errors but while looking at it, this came out as a way to have plunker examples be formatted in dev